module/files/helper.js

```diff
@@ -7,404 +7,444 @@
-     */
-
-    static saveZipFile(name, data) {
-        return new Promise(async (resolve, reject) => {
-            try {
-
-                if( !fs.existsSync(`${ROOT_PATH}${process.env.ZIP_PATH}`) ) {
-                    fs.mkdirSync(`${ROOT_PATH}${process.env.ZIP_PATH}`);
-                }
-                
-                let zipFileName = `${ROOT_PATH}${process.env.ZIP_PATH}/${name}`
-
-                fs.writeFile(zipFileName,data,async function(err){
-                    if(err) {
-                        return resolve({
-                            success : false
-                        })
-                    } else {
-                        return resolve({
-                            success : true
-                        })
-                    }
-                })
-
-            } catch (error) {
-                return reject(error);
-            }
+      } catch (error) {
+        return reject(error)
+      }
+    })
+  }
+
+  /**
+   * Save zip file in public zip folder
+   * @method
+   * @name saveZipFile
+   * @param  {String} zipFileName  - name of zip file.
+   * @param  {String}  zipFileData
+   * @return {Object} - Save zip file data.
+   */
+
+  static saveZipFile (name, data) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        if (!fs.existsSync(`${ROOT_PATH}${process.env.ZIP_PATH}`)) {
+          fs.mkdirSync(`${ROOT_PATH}${process.env.ZIP_PATH}`)
+        }
+
+        let zipFileName = `${ROOT_PATH}${process.env.ZIP_PATH}/${name}`
+
+        fs.writeFile(zipFileName, data, async function (err) {
+          if (err) {
+            return resolve({
+              success: false
+            })
+          } else {
+            return resolve({
+              success: true
+            })
+          }
         })
-    }
+      } catch (error) {
+        return reject(error)
+      }
+    })
+  }
+
+  /**
+   * Remove folder recursively
+   * @function
+   * @name removeFolder
+   * @param path - folder path.
+   * @returns {Promise}
+   */
 
-    /**
-     * Remove folder recursively
-     * @function
-     * @name removeFolder
-     * @param path - folder path.
-     * @returns {Promise}
-    */
+  static removeFolder (path) {
+    _removeFolder(path)
+    return
+  }
 
-    static removeFolder(path) {
-        _removeFolder(path);
-        return;
-    };
+  /**
+   * Save the file in cloud
+   * @method
+   * @name uploadToPresignedUrl
+   * @param  {String} url  - presignedUrl
+   * @param  {File}  file - file data
+   * @return {String} name - name of the file
+   */
 
-}
+  /**
+   * Upload file in different services based on cloud storage provided.
+   * @method
+   * @name upload
+   * @param  {file}  - file to upload.
+   * @param  {filePathForBucket}  - file path where the file should upload.
+   * @param {String} - bucketName
+   * @returns {json} Response consists of links of uploaded file.
+   */
 
+  static uploadFile (file, filePathForBucket, bucketName, storage) {
```

Comment:
@rakeshSgr - Let's have a common function for upload. Also, fix indentation so no other changes come

Reply:
resolved